### PR TITLE
Fix documentation examples for ch_types configuration option

### PR DESCRIPTION
**Pull request by allermat**
### Before merging …

- [x] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

**Comment by larsoner** [comment]
Can you try going to https://app.circleci.com/pipelines/github/mne-tools/mne-bids-pipeline/3751/workflows/61c987e8-e885-4433-9e0c-1e1014a8ab20 and then logging in with GitHub credentials to see if it solves the CircleCI issue? If not (or you don't want to) I can just push an empty commit on top of yours

**Comment by allermat** [comment]
@larsoner I just did that, not sure how to rerun the tests?

File: docs/source/v1.4.md.inc [hunk 1]
```diff
@@ -16,3 +16,4 @@
 
 - Fix bug when `mf_reference_run != runs[0]` (#742 by @larsoner)
 - Fix bug with too many JSON files found during empty room matching (#743 by @allermat)
+- Fix bug with outdated info on ch_types config option (#744 by @allermat)
```
Comment (larsoner) [inline docs/source/v1.4.md.inc:19]:
```suggestion
- Fix bug with outdated info on ch_types config option (#745 by @allermat)
```

**Comment by larsoner** [comment]
No idea what's going on with CircleCI now but I'll merge and look at how it goes on `main`. Thanks @allermat !

**Pull request merged**